### PR TITLE
add homescreen

### LINKED ISSUE #4: some stuff to be done
- [x] bugfixing
- [x] add home screen
- [x] add options

### PR DESCRIPTION
**Pull request by hRobert12**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/hRobert12/My-games/pull/13%23issuecomment-159097435%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/13%23issuecomment-160006087%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/13%23issuecomment-204611943%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/13%23issuecomment-204612043%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/13%23discussion_r45675709%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/13%23discussion_r45675917%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/13%23discussion_r45675952%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/13%23discussion_r45675975%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/13%23discussion_r45676925%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/13%23discussion_r45677019%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/13%23discussion_r45677045%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/13%23discussion_r45677528%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/13%23discussion_r45677546%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/13%23discussion_r58282579%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/13%23discussion_r58287455%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/13%23discussion_r58303313%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/hRobert12/My-games/pull/13%23issuecomment-159097435%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ca%20href%3D%5C%22https%3A//app.review.ninja/hRobert12/My-games/pull/13%5C%22%20target%3D%5C%22_blank%5C%22%3E%3Cimg%20src%3D%5C%22https%3A//app.review.ninja/46381900/pull/13/badge%5C%22%20alt%3D%5C%22ReviewNinja%5C%22/%3E%22%2C%20%22created_at%22%3A%20%222015-11-23T23%3A13%3A04Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/8714728%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/review-ninja%22%7D%7D%2C%20%7B%22body%22%3A%20%22This%20will%20close%20%234%20%22%2C%20%22created_at%22%3A%20%222015-11-26T23%3A42%3A24Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222016-04-01T23%3A55%3A58Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222016-04-01T23%3A56%3A45Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%202d1a9495ec4ac491177488ec42da27ceeabf6d69%20random_game.py%2016%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/hRobert12/My-games/pull/13%23discussion_r45675709%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22this%20will%20go%20to%20options%22%2C%20%22created_at%22%3A%20%222015-11-23T23%3A14%3A44Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%2C%20%7B%22body%22%3A%20%22%21fix%22%2C%20%22created_at%22%3A%20%222015-11-23T23%3A16%3A56Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-11-23T23%3A26%3A46Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A-1%3A%22%2C%20%22created_at%22%3A%20%222015-11-23T23%3A27%3A44Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-11-23T23%3A27%3A58Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A-1%3A%20%22%2C%20%22created_at%22%3A%20%222015-11-23T23%3A32%3A25Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-complete%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222016-04-01T23%3A55%3A58Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%2C%20%7B%22body%22%3A%20%22%21completed%22%2C%20%22created_at%22%3A%20%222016-04-02T03%3A04%3A16Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222016-04-03T02%3A35%3A44Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20random_game.py%3AL54-69%22%7D%2C%20%22Pull%202d1a9495ec4ac491177488ec42da27ceeabf6d69%20random_game.py%204%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/hRobert12/My-games/pull/13%23discussion_r45675952%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%21resolve%22%2C%20%22created_at%22%3A%20%222015-11-23T23%3A17%3A17Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%2C%20%7B%22body%22%3A%20%22%21fixed%22%2C%20%22created_at%22%3A%20%222015-11-23T23%3A17%3A32Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222015-11-23T23%3A32%3A35Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20random_game.py%3AL54-69%22%7D%7D%2C%20%22approved%22%3A%20%7B%22https%3A//github.com/hRobert12%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/hRobert12'></a>
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/hRobert12/My-games/pull/13#issuecomment-159097435'>General Comment</a></b>
- <a href='https://github.com/review-ninja'></a> <a href="https://app.review.ninja/hRobert12/My-games/pull/13" target="_blank">ReviewNinja
- <a href='https://github.com/hRobert12'></a> This will close #4
- [x] <a href='#crh-comment-Pull 2d1a9495ec4ac491177488ec42da27ceeabf6d69 random_game.py 16'></a> &nbsp;<b><a href='https://github.com/hRobert12/My-games/pull/13#discussion_r45675709'>File: random_game.py:L54-69</a></b>
- <a href='https://github.com/hRobert12'></a> this will go to options
- <a href='https://github.com/hRobert12'></a> !fix
- <a href='https://github.com/hRobert12'></a> :+1:
- <a href='https://github.com/hRobert12'></a> :-1:
- <a href='https://github.com/hRobert12'></a> :+1:
- <a href='https://github.com/hRobert12'></a> :-1:
- <a href='https://github.com/hRobert12'></a> :+1:<a href='#crh-update-complete'></a>
- <a href='https://github.com/hRobert12'></a> !completed
- [x] <a href='#crh-comment-Pull 2d1a9495ec4ac491177488ec42da27ceeabf6d69 random_game.py 4'></a> &nbsp;<b><a href='https://github.com/hRobert12/My-games/pull/13#discussion_r45675952'>File: random_game.py:L54-69</a></b>
- <a href='https://github.com/hRobert12'></a> !resolve
- <a href='https://github.com/hRobert12'></a> !fixed

<a href='https://www.codereviewhub.com/hRobert12/My-games/pull/13?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/hRobert12/My-games/pull/13?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/hRobert12/My-games/pull/13?approve=0'></a>&nbsp;<a href='https://github.com/hRobert12/My-games/pull/13'></a>
<a href='#crh-end'></a>

[Review on Reviewable](https://reviewable.io/reviews/hrobert12/my-games/13)

---

---

---

---

---

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/hRobert12/My-games/pull/13" target="_blank">ReviewNinja

**Comment by hRobert12** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/hRobert12/My-games/pull/13'>Refresh</a>.<a href='#crh-update-none'></a>

File: random_game.py [hunk 1]
```diff
@@ -54,7 +54,15 @@ def checknumber(number):
...
+            inp = inputnumber()
+            if checknumber(inp):
+                ntg = random.randint(minn, maxn)
+                trys = 0
+                break()
+            else:
+                trys = trys + 1
         get_inp = False
     elif inp == o
         """Options"""
```
Comment (hRobert12) [inline random_game.py]: this will go to options

Comment (hRobert12) [inline random_game.py]: !fix

File: random_game.py [hunk 2]
```diff
@@ -54,7 +54,15 @@ def checknumber(number):
     print("Type your pick in lower-case")
     inp = input("/n")
     if inp == p
-        """Play"""
```
Comment (hRobert12) [inline random_game.py]: !resolve

Comment (hRobert12) [inline random_game.py]: !fixed

File: random_game.py (hunk 1 above)
Comment (hRobert12) [inline random_game.py]: :+1:

Comment (hRobert12) [inline random_game.py]: :-1:

Comment (hRobert12) [inline random_game.py]: :+1:

Comment (hRobert12) [inline random_game.py]: :-1:

File: random_game.py (hunk 2 above)
Comment (hRobert12) [inline random_game.py]: :+1:

**Comment by hRobert12** [comment]
This will close #4

File: random_game.py (hunk 1 above)
Comment (hRobert12) [inline random_game.py]: :+1:<a href='#crh-update-complete'></a>

**Comment by hRobert12** [comment]
:+1:<a href='#crh-update-none'></a>

**Comment by hRobert12** [comment]
:shipit:

File: random_game.py (hunk 1 above)
Comment (hRobert12) [inline random_game.py]: !completed

Comment (hRobert12) [inline random_game.py]: :+1:

**Pull request merged**